lib/migration.pm

```diff
@@ -158,9 +158,13 @@ sub disable_installation_repos {
 
 # Based on bsc#1097111, need to disable kernel multiversion before migration, and enable it after migration
 # https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-update-preparation.html#sec-update-preparation-multiversion
-sub disable_kernel_multiversion {
-    my $sed_para = check_var('VERSION', get_var('ORIGIN_SYSTEM_VERSION')) ? 's/^multiversion/#multiversion/g' : 's/^#multiversion/multiversion/g';
+# Acceplable arguements "disable" e.g. modify_kernel_multiversion("disable");
+# Any other arguement will enable multiversion kernel.
+sub modify_kernel_multiversion {
```

Comment:
Please don't remove the condition for continuous migration, this enable/disable multiversion is not required for normal migration test, and this is the workaround for bsc#1097111 which  only happened on continuous migration test.  If you really want to remove the condition, then need run enough VRs and clear document support.

Reply:
I didn't remove any condition, I just didn't move it into the function as Yiannis proposed, because , now the function is not migration specific anymore.

Reply:
Aha, I check it again, right, then no question from my side.